doc/association-rules.rst

```diff
@@ -0,0 +1,74 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+Data set
```

Comment:
Suppose there were multiple input signals. Consider the generated HTML difference given the following two inputs:
```rst
Signals
-------
**Inputs**:

- **Data**

Data set

- **Subset data**

Data subset
```
vs.
```rst
Signals
-------
#### Inputs

- **Data**
   
  Data set

- **Subset data**
   
  Data subset
```

The first translates into:
```html
<h2>Signals</h2>
<p><strong>Inputs</strong>:</p>
<ul>
<li><strong>Data</strong></li>
</ul>
<p>Data set</p>
<ul>
<li><strong>Subset data</strong></li>
</ul>
<p>Data subset</p>
```
The second into:
```html
<h2>Signals</h2>
<h4>Inputs</h4>
<ul>
<li><p><strong>Data</strong></p>
    <p>Data set</p></li>
<li><p><strong>Subset data</strong></p>
    <p>Data subset</p></li>
</ul>
```
One is much better than the other.